openmc/deplete/operator.py

```diff
@@ -198,10 +220,16 @@ def _get_burnable_mats(self):
             Nuclides in order of how they'll appear in the simulation.
 
         """
+
+        # Automatically distribute burnable materials
+        self._differentiate_burnable_mats()
```

Comment:
An optional argument was added so that the user can keep burnable materials over multiple instances to be burned as one region. But in this case, the user is responsible for giving a correct volume of the material (it is the total volume of all the instances).